spec/controllers/coronavirus_landing_page_controller_spec.rb

```diff
@@ -12,7 +13,32 @@
 
     it "has a success response" do
       get :show
+
       expect(response).to have_http_status(:success)
     end
+
+    context "User Experience Measurement practice AB test" do
+      render_views
+      it "should render the A version of the page" do
+        with_variant UxmPracticeTest: "A" do
+          get :show
+          expect(response).to have_http_status(:success)
+          expect(response.header["Vary"]).to eq("GOVUK-ABTest-UxmPracticeTest")
+          expect(response.body).to include("<meta name=\"govuk:ab-test\"  content=\"UxmPracticeTest:A\"  data-analytics-dimension=\"68\"  data-allowed-variants=\"A,B,Z\">")
+        end
+      end
+
+      it "should render the B version of the page" do
+        with_variant UxmPracticeTest: "B" do
+          get :show
+          expect(response).to have_http_status(:success)
+          expect(response.header["Vary"]).to eq("GOVUK-ABTest-UxmPracticeTest")
+          assert_equal("GOVUK-ABTest-UxmPracticeTest", response.headers["Vary"])
+          expect(response.body).to include(
+            "<meta name=\"govuk:ab-test\"  content=\"UxmPracticeTest:B\"  data-analytics-dimension=\"68\"  data-allowed-variants=\"A,B,Z\">",
+          )
+        end
+      end
```

Comment:
I guess the `Z` variant is covered by the normal tests that just result in the `show` view being rendered. It might be nice to add it in here for explicitness though?

Reply:
Also possibly worth testing what happens if the supplied header is not A,B or Z. Just in case something goes wonky. You know I hate wonky

Reply:
Thanks Simon! I have added these